.github/workflows/gaia.yml

```diff
@@ -65,6 +65,11 @@ jobs:
         cd ${GITHUB_WORKSPACE}/research/query_service/ir/compiler
         make build
 
+    - name: Build Ir on Vineyard Store
+      run: |
+        source ${HOME}/.bashrc
```

Comment:
Why do we need to source this bashrc? It is user's customed envs. 
If you have envs, should put them together into the [.graphscope_env](https://github.com/alibaba/GraphScope/blob/main/scripts/install_deps.sh#L20)

Reply:
I'm not sure why, but we do have many `source ~/.bashrc` lines across all CI yamls. It might be used for leveraging local build cache or something like that, I guess.